api/common/src/main/java/io/opentelemetry/api/common/ArrayBackedAttributesBuilder.java

```diff
@@ -98,23 +50,35 @@ public AttributesBuilder putAll(Attributes attributes) {
     return this;
   }
 
-  private static List<Double> toList(double... values) {
+  @Nullable
+  static List<Double> toList(double... values) {
```

Comment:
I think we should either mark the `double...` as `@Nullable` or drop the `@Nullable` from the return type annotation. Or we could actually leave the code as it was here, or return `Collections.emptyList()`.

Reply:
yeah, we're already guarding all the calls so they don't pass in `null` here, so I think we can just return emptyList if someone figures out a way around it.

Reply:
Agree it's hard enough to pass null so anything is fine including empty list.

Reply:
Actually I went with null (and added `@Nullable`) to the parameter. Otherwise the repeated values would have different behavior on null vs the singular values, didn't see a good enough reason to do that.